packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -334,6 +361,20 @@ export default function PostOptionsMenu({
     });
   }
 
+  if (isLoggedIn && isSourceSubscribeV1 && !isSourceBlocked) {
+    postOptions.push({
+      icon: (
+        <MenuIcon
+          Icon={sourceSubscribe.isSubscribed ? BellSubscribedIcon : BellIcon}
+        />
+      ),
+      label: sourceSubscribe.isSubscribed
+        ? `Unsubscribe from ${post?.source?.name}`
+        : `Subscribe to ${post?.source?.name}`,
```

Comment:
The suffix with name seems to be the same in both cases, we could do the string literal in a different way:
```suggestion
      label: `${sourceSubscribe.isSubscribed ? 'Unsubscribe from' : 'Subscribe to'} ${post?.source?.name}`,
```

Reply:
Yes!